erpnext_com/www/docs/user/manual/en/erpnext_integration/contents.md

```diff
@@ -6,28 +6,29 @@
 
```

Comment:
@barredterra why mark as 'Only Backup'? Being a cloud file storage service, do you see any other use for Google Drive integrated with ERPNext?
Numbering the bullets is unnecessary, markdown will automatically render numbers in ascending order.

Reply:
Yes, you could store all user files in Google Drive or have the option to upload them there, or provide access to files from Google Drive.

I find it nice to be able to read the text unrendered.